docs/user-guide/src/basic.md

```diff
@@ -15,15 +15,21 @@ See the [following chapter](verify/summary.md) for a list of verification featur
 
 To run Prusti on a file using the command-line setup:
 
-```bash
-$ prusti-rustc --edition=2018 path/to/file.rs
+```sh
+prusti-rustc --edition=2018 path/to/file.rs
+```
+
+To run Prusti on a Rust crate:
```

Comment:
- "crate *or workspace*"
- Add a note that this command should be invoked from within the crate/workspace root.
- If `cargo-prusti` is in `PATH`, then I think you can run `cargo prusti` (i.e. it behaves like `clippy` or any other third-party `cargo` command).